charts/kyverno/templates/_helpers.tpl

```diff
@@ -68,6 +68,15 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- printf "%s" (default (printf "%s-metrics" (include "kyverno.fullname" .)) .Values.config.existingMetricsConfig) -}}
 {{- end -}}
 
+{{/* Get the pull secret list. */}}
+{{- define "kyverno.pullsecretlist" -}}
+{{- $pullsecrets := list -}}
+{{- range .Values.existingImagePullSecrets -}}
+{{-   $pullsecrets = . | append $pullsecrets -}}
+{{- end }}
+{{- join "," (concat (keys .Values.imagePullSecrets) $pullsecrets) -}}
```

Comment:
Why take the array and regenerate another array that is presumably the exact same thing?  Why not try this:

```
{{- join "," (concat (keys .Values.imagePullSecrets) .Values.existingImagePullSecrets) -}}
```

Reply:
Great suggestion - that tested out properly as well.